tests/CMakeLists.txt

```diff
@@ -21,16 +26,15 @@ target_link_libraries(models_test
   ${ARMADILLO_LIBRARIES}
   ${Boost_FILESYSTEM_LIBRARY}
   ${Boost_SYSTEM_LIBRARY}
-  ${Boost_SERIALIZATION_LIBRARY}
   ${Boost_REGEX_LIBRARY}
   ${MLPACK_LIBRARIES}
-)
-
-# Add tests to the testing framework.
-# Get the list of sources from the test target.
-get_target_property(test_sources models_test SOURCES)
+)  
 
 # So the dll is placed in the same dir as the tests.
 set(CMAKE_RUNTIME_OUTPUT_DIRECTORY ${CMAKE_BINARY_DIR}/bin)
 
-add_test(NAME "test" COMMAND models_test)
+if(NOT WIN32)
+  catch_discover_tests(models_test)
+else()
+  add_test(NAME "test" COMMAND models_test)
+endif()
```

Comment:
If you don't mind making sure that the file ends with a newline, I'd appreciate it. :+1:

Reply:
I generally keep that in mind but somehow missed it. Sorry for that.